hever/inc/customize-preview-wpcom.js

```diff
@@ -0,0 +1,103 @@
+/**
```

Comment:
I think this file should be renamed to `wpcom-customize-preview.js` for consistency.

Reply:
I went for this because varia, and other default themes use the same naming convention (https://github.com/Automattic/themes/blob/trunk/varia/inc/customize-preview-wpcom.js), it's just seedlet that does it differently

Reply:
Since a lot of this is going to be shared with other child themes can we put it as a lib in varia?

Reply:
> Since a lot of this is going to be shared with other child themes can we put it as a lib in varia?

This file already exists for Varia and if we add it right now it will be adding those css variables to varia too, which may affect Varia in some way, since the css variables have the same as the ones gutenberg creates (not a 100% sure on how that would affect it). I'd say do this on a separate PR that migrates Varia to the new system too, to be sure we aren't breaking stuff and allow for the rest of the children to inherit everything as well

Reply:
I opened the issue for it and added to the board: https://github.com/Automattic/themes/issues/3038